### ISSUE TITLE
Deploy demo of Datasette on every commit that passes tests

### ISSUE THREAD

**Issue by simonw** [simonw/datasette#313]
We can use Travis CI and Zeit Now to ensure there is always a live demo of current master. We can ship archived demos for releases as well.

**Comment by simonw** [comment]
This very nearly works...

* https://latest.datasette.io/
* https://f0c1722.datasette.io/

But... https://f0c1722.datasette.io/-/versions isn't showing the correct note:
```
{
    "datasette": {
        "version": "0.22.1"
    }
    ...
```
There should be a `"note"` field there with the full commit hash.

**Comment by simonw** [comment]
It looks like all of my test deploys ended up going to the same Zeit deployment ID: https://zeit.co/simonw/datasette-latest/rbmtcedvlj

This is strange... the Dockerfile should be different for each one (due to the differing version-note).

https://github.com/simonw/datasette/commit/db1e6bc182d11f333e6addaa1a6be87625a4e12b#diff-34418c57343344c73271e13b01b7fcd9R255

**Comment by simonw** [comment]
```
The command "datasette publish now fixtures.db -m fixtures.json --token=$NOW_TOKEN --branch=$TRAVIS_COMMIT --version-note=$TRAVIS_COMMIT" exited with 0.
```

Partial log of the ``datasette publish now`` output:

```
> Step 5/7 : RUN datasette inspect fixtures.db --inspect-file inspect-data.json
>  ---> Running in d373f330e53e
>  ---> 09bab386aaa3
> Removing intermediate container d373f330e53e
> Step 6/7 : EXPOSE 8001
>  ---> Running in e0fe37b3061c
>  ---> 47798440e214
> Removing intermediate container e0fe37b3061c
> Step 7/7 : CMD datasette serve --host 0.0.0.0 fixtures.db --cors --port 8001 --inspect-file inspect-data.json --metadata metadata.json --version-note f0c17229b7a7914d3da02e087dfd0e25d8321448
```

So it looks like `--version-note` is being correctly set there.

**Comment by simonw** [comment]
Aha! ```1.03s$ now alias --token=$NOW_TOKEN
> Error! Couldn't find a deployment to alias. Please provide one as an argument.
The command "now alias --token=$NOW_TOKEN" exited with 1.```
That explains it. I need to set the same alias in my call to `datasette publish`.

**Comment by simonw** [comment]
That fixed it! https://958b75c.datasette.io/-/versions

```
{
  "python": {
    "version": "3.6.5",
    "full": "3.6.5 (default, Jun  6 2018, 19:19:24) \n[GCC 6.3.0 20170516]"
  },
  "datasette": {
    "version": "0+unknown",
    "note": "958b75c69841ef5913da86e0eb2df634a9b95fda"
  },
  "sqlite": {
    "version": "3.16.2",
    "fts_versions": [
      "FTS5",
      "FTS4",
      "FTS3"
    ],
    "extensions": {
      "json1": null
    }
  }
}
```
